README.md

```diff
@@ -51,6 +51,12 @@ This project is configured using environment variables.
 |                       | `OAUTH_CLIENT_SECRET`        | Oauth client secret                                                                                                                                                                                                                  | _empty string_                  |
 |                       | `OAUTH_TOKEN_URL`            | Observation portal Oauth token URL                                                                                                                                                                                                   | `http://localhost/o/token/`     |
 |                       | `OAUTH_PROFILE_URL`          | Observation portal profile URL                                                                                                                                                                                                       | `http://localhost/api/profile/` |
+| Appearance Settings   | `NAVBAR_TITLE_TEXT`          | Name that appears in the navbar of the browsable api                                                                                                                                                                                 | `Science Archive API`           |
+|                       | `NAVBAR_TITLE_URL`           | Hyperlink for the NAVBAR_TITLE_TEXT                                                                                                                                                                                                  | `https://archive.lcogt.net`     |
+|                       | `PAGINATION_DEFAULT_LIMIT`   | Numeric value indicating the page size for results                                                                                                                                                                                   | `100`                           |
+| More customization    | `ZIP_DOWNLOAD_FILENAME_BASE` | Initial part of the zip download filename                                                                                                                                                                                            | `ocs_archive_data`              |
+|                       | `ZIP_DOWNLOAD_MAX_UNCOMPRESSED_FILES`     | Maximum number of files that users can bundle in a single uncompressed zipped download                                                                                                                                  | `10`                            |
+|                       | `TERMS_OF_SERVICE_URL`       | URL pointing to a terms of service for users                                                                                                                                                                                         | `https://lco.global/policies/terms/` |
```

Comment:
I'm not super clear on how the terms are used so maybe there is a better way to describe this.